javascript-forms/form-values.js

```diff
@@ -0,0 +1,13 @@
+var $forms = document.querySelector('#contact-form');
+
+$forms.addEventListener('submit', function (event) {
+  event.preventDefault();
+
+  console.log('messageData:', {
```

Comment:
make sure you are actually creating the object then logging it out.

Reply:
https://user-images.githubusercontent.com/67080126/141912518-296a4e7d-a1b1-4c9a-8773-3b093af8c832.mov